swan-lake/integration-tutorials/sending-emails-from-a-service.md

```diff
@@ -99,158 +99,108 @@ Follow the instructions given in this section to develop the service.
     $ bal new sending-emails-from-a-service
     ```
 
-2. Introduce the source code in files with the `.bal` extension (e.g., the `main.bal` file). 
+2. Remove the generated content in the `main.bal` file and open the diagram view in VS Code.
 
-    Import the 
-    - `ballerina/email` module to send emails using the SMTP protocol
-    - `ballerina/http` module to develop the REST API and define the clients that can be used to send requests to the backend services
-    - `ballerina/log` module to log debug, error or info level information for each client request
+    ![Open diagram view](/learn/images/integration-tutorials/sending-emails-from-a-service/open_diagram_view.gif)
 
-    ```ballerina
-    import ballerina/email;
-    import ballerina/http;
-    import ballerina/log;
+3. Generate a record type corresponding to the payload from the backend service by providing a sample of the expected JSON payload.
+
+    The payload from the backend service will be a JSON object similar to the following.
+
+    ```json
+    {
+        "appointmentNumber": 1,
+        "doctor": {
+            "name": "thomas collins",
+            "hospital": "grand oak community hospital",
+            "category": "surgery",
+            "availability": "9.00 a.m - 11.00 a.m",
+            "fee": 7000.0
+        },
+        "patient": {
+            "name": "John Doe",
+            "dob": "1940-03-19",
+            "ssn": "234-23-525",
+            "address": "California",
+            "phone": "8770586755",
+            "email": "johndoe@gmail.com"
+        },
+        "hospital": "grand oak community hospital",
+        "confirmed": false,
+        "appointmentDate": "2023-10-02"
+    }
     ```
 
-3. Define [configurable variables](https://ballerina.io/learn/by-example/#configurability) for the port on which the listener should listen and the URLs of the backend services. Also, define configurable variables for the host, username, and password of the SMTP client.
+    ![Define records](/learn/images/integration-tutorials/sending-emails-from-a-service/define_records.gif)
 
-    > **Note:** Enable two-factor authentication on your Google account, generate an app password, and use the app password in place of your email password. The app password can be generated with [this link](https://myaccount.google.com/apppasswords?pli=1&rapt=AEjHL4Mf5XDD4rE79YJpP5E2NoNwhvXMET_TWyBcQRn-HMzt0PI8BmptpMGRiBVIamW-0ECgVZtXxMRA19bL4Wfnq_hmjBEMqA).
+    Similarly, generate record types for other payloads from the backend service.
```

Comment:
Mention which backend specifically, since there are only two.

Reply:
Removed the `Similarly..` part